WalletWasabi.Fluent/Models/Pocket.cs

```diff
@@ -20,4 +21,16 @@ public Pocket((SmartLabel labels, ICoinsView coins) pocket)
 	public ICoinsView Coins { get; }
 
 	public static Pocket Empty => new((SmartLabel.Empty, new CoinsView(Enumerable.Empty<SmartCoin>())));
+
+	public Money EffectiveSumValue(FeeRate feeRate) => Coins.Sum(coin => coin.EffectiveValue(feeRate));
+
+	public static Pocket operator +(Pocket x, Pocket y)
+	{
+		var mergedLabels = SmartLabel.Merge(x.Labels, y.Labels);
+		var mergedCoins = new CoinsView(x.Coins.Concat(y.Coins));
```

Comment:
I think this is not good, because `CoinsView` doesn't ensure uniqueness

```suggestion
		var mergedCoins = new CoinsView(x.Coins.Concat(y.Coins).ToHashSet());
```

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/9126/commits/d02dcc76efcf2e1c29616ea5e979532567520ec1